docs/developer-guide/test-and-debug/processing-log.md

```diff
@@ -121,6 +121,11 @@ message.deserializationError (STRUCT)
 :   The contents of a message with type 0 (DESERIALIZATION_ERROR).
     Logged when a deserializer fails to deserialize an {{ site.ak }} record.
 
+message.deserializationError.component (STRING)
```

Comment:
Not a huge fan of the name "component" but couldn't think of anything better. Hopefully someone else has a better suggestion?

Reply:
I poked around a bit and didn't see consensus on a good term, but I did see a few sources using the very technical term "part", like "key-part" and "value-part".

Reply:
Don't mind component or part, or how about _target_?

Reply:
Sure, updated to "target".